docs/migrating.md

```diff
@@ -100,14 +100,18 @@ code, you may need to fix your tests.
 
 Some likely issues include:
 
-* `android.view.InflateException: Binary XML file line #3: Failed to resolve attribute at index 17:
-  TypedValue{t=0x2/d=0x7f01000e a=-1}`<br/>
-  This happens when your [`Activity`][activity-documentation] is using a theme that lacks values for
-  certain attributes used by layouts. Make sure you've specified an appropriate theme for your
-  activities in your `AndroidManifest`.
+!!! quote ""
```

Comment:
The linter was complaining that we use a `<br/>` HTML tag. So I changed the layout to something different:

<img width="723" alt="Screenshot 2024-10-27 at 17 53 34" src="https://github.com/user-attachments/assets/8b296a74-9961-46b8-bd48-098297fadb54">